lib/views/create_livestream/active_stream.dart

```diff
@@ -105,7 +153,7 @@ class _Backdrop extends StatelessWidget {
     return Hero(
       tag: 'go live',
       child: Transform.scale(
-        scale: 1.1,
+        scale: 1.25,
```

Comment:
In order for the "Go Live" button animation to work, the background needs to be its own widget. This scaling helps make sure it covers the whole screen.